src/apps/migrate/run.py

```diff
@@ -413,6 +417,29 @@ async def migrate_alerts(
     await AlertMigrationService(session, alerts).run_alerts_migration()
 
 
+async def migrate_pending_invitations(
+    applet_ids: list[ObjectId] | None, mongo: Mongo, postgres: Postgres
+):
+    invitations_collection = mongo.db["invitation"]
+    session = session_manager.get_session()
+
+    query = {}
+    if applet_ids:
+        query["appletId"] = {"$in": applet_ids}
+
+    invitations: list = []
+    for invitation in invitations_collection.find(query):
+        invitations.append(MongoInvitation.parse_obj(invitation))
```

Comment:
As far as I remember, our role names do not match, in the old system the user is equal to the respondent in the new system, please check this point

Reply:
done.